src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,15 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns a modifiable java List view.
+     * For use with other methods/libraries.
+     * Any changes to the internal list/elements are immediately visible in the returned list.
+     */
+    public List<Person> mutableListView() {
+        return internalList;
```

Comment:
Hmm, do you think the method name is accurate considering you are exposing the internal list which was previously abstracted behind another method?